validator/src/main/java/com/amazon/aoc/validators/PerformanceValidator.java

```diff
@@ -25,6 +25,7 @@
   private static int MAX_RETRY_COUNT = 30;
   private Context context;
   private ValidationConfig validationConfig;
+  private static final int bytesInMegaBytes = 1000000;
```

Comment:
nitpick: Move up to line 24.
```suggestion
  private static final int BYTES_IN_MEGABYTES = 1000000;
```